docs/overrides/home.html

```diff
@@ -27,8 +27,12 @@
                             </p>
                         </div>
                         <div class="flex flex-col items-center justify-center gap-3 mt-10 lg:flex-row lg:justify-start">
+                            <a href="https://2b0ac664.sibforms.com/serve/MUIFAGN41TZby6SgxI7eQyUevVgGPjMujrCwZu7plP3WOxGbvBxgP2rFCxF6dPKNe-l8IoczAYTKkSmXmDSR9_NOfAe1RIf1nbfqExLbji7XKd4bQcptpG7PHwHQU5Y1-PGw2EkqxX7nFuOh0tcbAMZ1Jbs2gh02J8-ya9bIjuLzfdfi0kdkESFrDw2rt7BWMEwJRQd9DfGzU016"
+                                class="items-center justify-center w-full px-6 py-2.5 text-center text-white duration-200 bg-gradient-to-bl from-fuchsia-600 to-orange-300 border-2 border-white  rounded-full inline-flex hover:bg-transparent hover:border-black hover:text-black text-sm lg:w-auto">
+                                Get in touch
```

Comment:
Since the mailing list is mostly intended for announcements/updates and less for active communication with us (for which I'd say GH issues/Discord are our main avenues), "Get in touch" is not my favourite title here, especially since we also have our dedicated Getting Help page as well.

How about something like "Stay updated" or "Join our update list" instead? I think users will also be more inclined to click the button if it doesn't imply that they aren't required to reach out to us first, rather that we will communicate with them just by virtue of them joining the list.